rerun_js/web-viewer/index.ts

```diff
@@ -496,6 +519,144 @@ export class WebViewer {
     }
   }
 
+  /**
+   * Get the active recording id.
+   */
+  get_active_recording_id(): string | null {
+    if (!this.#handle) {
+      throw new Error(
+        `attempted to get active recording id in a stopped web viewer`,
+      );
+    }
+
+    return this.#handle.get_active_recording_id() ?? null;
+  }
+
+  /**
+   * Set the active recording id.
+   */
+  set_active_recording_id(value: string) {
+    if (!this.#handle) {
+      throw new Error(
+        `attempted to set active recording id to ${value} in a stopped web viewer`,
+      );
+    }
+
+    this.#handle.set_active_recording_id(value);
+  }
+
+  /**
+   * Get the play state.
+   *
+   * This always returns `false` if the recording can't be found.
+   */
+  get_playing(recording_id: string): boolean {
+    if (!this.#handle) {
+      throw new Error(`attempted to get play state in a stopped web viewer`);
+    }
+
+    return this.#handle.get_playing(recording_id) || false;
+  }
+
+  /**
+   * Set the play state.
+   *
+   * This does nothing if the recording can't be found.
+   */
+  set_playing(recording_id: string, value: boolean) {
+    if (!this.#handle) {
+      throw new Error(
+        `attempted to set play state to ${
+          value ? "playing" : "paused"
+        } in a stopped web viewer`,
+      );
+    }
+
+    this.#handle.set_playing(recording_id, value);
+  }
+
+  /**
+   * Get the current time.
+   *
+   * This always returns `0` if the recording or timeline can't be found.
+   */
+  get_current_time(recording_id: string, timeline: string): number {
```

Comment:
We should explain the unit here and in `set_current_time` (nanoseconds for temporal timelines)